### PR TITLE
KK-1218 | Logout idle user after timeout

### PR DESCRIPTION
**Pull request by melniiv**
## Context

[KK-1218](https://helsinkisolutionoffice.atlassian.net/browse/KK-1218)

## Screenshots

[KK-1218]: https://helsinkisolutionoffice.atlassian.net/browse/KK-1218?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr591.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kukkuu-ui-pr591.dev.hel.ninja :laughing::tada::tada::tada:

File: .env.development.local.example [hunk 1]
```diff
@@ -10,3 +10,4 @@ VITE_OIDC_AUDIENCES=kukkuu-api-dev,profile-api-test
 VITE_ADMIN_TICKET_VALIDATION_URL=https://kukkuu-admin-ui.test.hel.ninja/check-ticket-validity
 VITE_CMS_URI=https://kukkuu.app-staging.hkih.hion.dev/graphql
 VITE_HELSINKI_PROFILE_URL=https://profiili.test.hel.ninja/loginsso
+VITE_IDLE_TIMEOUT_IN_MS=3600000
```
Comment (karisal-anders) [inline .env.development.local.example:13]:
```suggestion
VITE_IDLE_TIMEOUT_IN_MS=3600000  # 60min = 60 * 60 * 1000ms
```

Comment (karisal-anders) [reply .env.development.local.example:13]: And similarly some comments in other places would be good IMHO for clarity.

Comment (melniiv) [reply .env.development.local.example:13]: comments in which places? I mean, for me it looks more clear as i ve done that, so if you point out what to comment, i will definitely do. 3600000 value we had already without a comment in other variable in use, so i thought its ok

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr591.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr591.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr591.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr591.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kukkuu-ui-pr591.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr591.dev.hel.ninja :crying_cat_face::anger::boom::boom:

File: src/domain/auth/IdleTimerProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    if (isAuthenticated()) {
+      logout();
+    }
+  };
+
+  return (
+    <IdleTimerProvider
+      timeout={AppConfig.userIdleTimeoutInMs || 3_600_000}
+      onIdle={onIdle}
+      name="att-kukku-ui-idle-timer"
```
Comment (karisal-anders) [inline src/domain/auth/IdleTimerProvider.tsx:21]: What does `att` mean in this context?

How about this?
```suggestion
      name="kukkuu-ui-idle-timer"
```

**Review by karisal-anders — APPROVED** [review]

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr591.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr591.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Pull request merged**